stable/postgresql/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 7.7.1
+version: 7.7.2
```

```suggestion
version: 8.0.0
```
as folks probably depend on the service-names.

---

I agree.
We also need to ensure this PR isn't including any incompatibility with previous versions, and, if that's case, we need to provide the corresponding instructions to upgrade the chart.

---

Thanks for looking at the PR! You're right, some people might depend on the port names outside of the dependencies in the chart itself. I'll bump the version and add a note in the docs to advise people to check dependencies on port names and change them accordingly.